bridge-token-factory/src/lib.rs

```diff
@@ -107,6 +160,16 @@ pub struct BridgeTokenFactory {
     /// Mask determining all paused functions
     #[deprecated]
     paused: Mask,
+    /// Fee amount bound for deposit.
+    pub deposit_fee_bound: UnorderedMap<String, DepositTokenBounds>,
+    /// Fee amount bound for withdraw
+    pub withdraw_fee_bound: UnorderedMap<String, WithdrawTokenBounds>,
+    /// Fee percentage of each token for deposit
+    pub deposit_fee_percentage: UnorderedMap<String, DepositFeePercentage>,
+    /// Fee percentage of each token for withdraw
+    pub withdraw_fee_percentage: UnorderedMap<String, WithdrawFeePercentage>,
```

Comment:
We could combine the `deposit_fee_bound` with `deposit_fee_percentage`, and `withdraw_fee_bound` with `withdraw_fee_percentage`. Because as I see, those values are used together in one call.

Reply:
Can we include this in separate PR, as this will change the code a lot? Please let me know, I can start accordingly

Reply:
I think it is better to refactor it in this pr, so you just need to add an additional structs that will contain both `XxFeePercentage` and `FeeBounds `

Reply:
2e8a52d8469f1efb6fe551eb8eb3e5ee079df585